diff --git a/node_modules/react-native-nodemediaclient/android/build.gradle b/node_modules/react-native-nodemediaclient/android/build.gradle
index 9762d19..ee98a5f 100644
--- a/node_modules/react-native-nodemediaclient/android/build.gradle
+++ b/node_modules/react-native-nodemediaclient/android/build.gradle
@@ -1,12 +1,12 @@
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion 33
 
 
     defaultConfig {
-        minSdkVersion 16
-        targetSdkVersion 28
+        minSdkVersion 24
+        targetSdkVersion 33
         versionCode 1
         versionName "1.0"
 
